arm/Microsoft.KeyVault/vaults/deploy.bicep

```diff
@@ -161,13 +158,11 @@ var maxNameLength = 24
 var uniquenameUntrim = uniqueString('Key Vault${baseTime}')
 var uniquename = (length(uniquenameUntrim) > maxNameLength ? substring(uniquenameUntrim, 0, maxNameLength) : uniquenameUntrim)
 var name_var = !empty(name) ? name : uniquename
```

Comment:
It appears the readme's still require the parameter table updates. Can you please rerun the `Set-ModuleReadme` script for the updates templates?

Reply:
done!